gems/smithy/lib/smithy/vise/endpoints/built_in_binding.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Vise
+    module Endpoints
+
+      # Endpoint BuiltInBinding
+      class BuiltInBinding
+        def initialize(id:, render_config:, render_build:, render_test_set:)
```

Comment:
I've just been using options = {} everywhere because of how easy it is to just pass options down the line without extracting keyword args... Not sure how you feel about that. I think I lean towards keyword args for required public interfaces.

Reply:
Yeah - I think for required public interfaces I'd generally prefer keyword args.  Since this is an extension point/public interface, I think keeping them as keywords make sense.

Reply:
I don't think this is public at all. Please just use options for vise usage. But I think these are better used as view classes now - since vise does not have any shape wrapping.

Reply:
These must be public - these are the extension point for users to add endpoint related bindings.

Reply:
I've removed the classes and just using arrays/hashes.